src/main/webapp/user-page.html

```diff
@@ -42,11 +42,19 @@
   </head>
   <body onload="initiateUI();">
   <!-- Website title header -->
-    <h1 class="site-heading text-center text-white d-none d-lg-block">
+    <h1 class="site-heading text-center text-white d-none d-lg-block mb-1">
```

Comment:
Is there a reason for this change? Does it make it inconsistent with the other pages?